File: reverse-word/ReverseWord.cs
public class ReverseWord
{
    public static string reverseWord(string word) 
    { 
        string[] words = word.Split(' ');
        for(int i = 0; i < words.Length / 2; i++) {
            int lastIndex = words.Length - i - 1;
            string temp = words[lastIndex];
            words[lastIndex] = words[i];
            words[i] = temp;
        }
        return string.Join(' ', words);
    }
}

File: reverse-word/ReverseWord.java
public class ReverseWord {
    public static String reverseWord(String word) {
        String[] words = word.split(" ");
        for(int i = 0; i < word.length(); i++) {
            int lastIndex = word.length() - i - 1;
            String temp = words[i];
            words[i] = words[lastIndex];
            words[lastIndex] = temp;
        }
        return String.join(" ", words);
    }
}